src/managers/input/Touch.cpp

```diff
@@ -24,6 +28,26 @@ void CInputManager::onTouchDown(wlr_touch_down_event* e) {
         return;
     }
 
+    // TODO: Don't swipe if you touched a floating window.
+    if (**PSWIPETOUCH && (!m_pFoundLSToFocus || m_pFoundLSToFocus->layer <= ZWLR_LAYER_SHELL_V1_LAYER_BOTTOM) && (!m_sActiveSwipe.pWorkspaceBegin)) {
+        const auto PWORKSPACE = g_pCompositor->getWorkspaceByID(PMONITOR->activeWorkspace);
+        const bool VERTANIMS  = PWORKSPACE->m_vRenderOffset.getConfig()->pValues->internalStyle == "slidevert" ||
+            PWORKSPACE->m_vRenderOffset.getConfig()->pValues->internalStyle.starts_with("slidefadevert");
+        // TODO: support no_gaps_when_only?
+        const auto TARGETPORTION = (**PGAPSOUT + **PBORDERSIZE) / (VERTANIMS ? PMONITOR->vecSize.y : PMONITOR->vecSize.x);
+        const auto POSITION      = (VERTANIMS ? e->y : e->x);
+        if (POSITION < TARGETPORTION || POSITION > (1 - TARGETPORTION)) {
```

Comment:
This logic is broken (since the rebase) -- seems to always return true

Reply:
Doing some printf debugging:
```
num: 187650383698822 / 404.000000TARGETPORTION: 464481147769.361389
POSITION: 0.650000
```

The problem is that `(**PGAPSOUT + **PBORDERSIZE)` returns *reallybignumber*

Reply:
I guess PGAPSOUT is a special data type instead of just an integer